eva/models/storage/batch.py

```diff
@@ -244,15 +244,16 @@ def project(self, cols: None) -> Batch:
         We do a copy for now.
         """
         cols = cols or []
-        verfied_cols = [c for c in cols if c in self._frames]
-        unknown_cols = list(set(cols) - set(verfied_cols))
-        if len(unknown_cols):
-            logger.warn(
-                "Unexpected columns %s\n\
-                                 Frames: %s"
-                % (unknown_cols, self._frames)
-            )
-        return Batch(self._frames[verfied_cols])
+        
+        # verfied_cols = [c for c in cols if c in self._frames]
```

Comment:
Add note explaining why code is commented out.

Reply:
Removed the commented block as its not needed.